core/internal/features/ocr2/features_ocr2_test.go

```diff
@@ -722,4 +738,28 @@ juelsPerFeeCoinSource = """
 	assert.Equal(t, digestAndEpoch.Epoch, epoch)
 }
 
+func generateDefaultOCR2OnchainConfig(minValue *big.Int, maxValue *big.Int) []byte {
```

Comment:
Could use a `T` here too?
```suggestion
func generateDefaultOCR2OnchainConfig(t *testing.T, minValue *big.Int, maxValue *big.Int) []byte {
```